app/controllers/oaisys/pmh_controller.rb

```diff
@@ -20,6 +20,8 @@ def bad_verb
   end
 
   def identify
+    expect_args
```

Comment:
Do you want to just stub out a simple `expect_no_args` method that does nothing but call exect_args?

It's functionally equivalent, but it might be a little more self-documenting.

Reply:
Makes sense, done!